massa-protocol-worker-2/src/handlers/block_handler/mod.rs

```diff
@@ -25,90 +35,40 @@ pub struct BlockHandler {
 impl BlockHandler {
     pub fn new(
         active_connections: SharedActiveConnections,
-        receiver: Receiver<(PeerId, u64, Vec<u8>)>,
-        receiver_ext: Receiver<BlockHandlerCommand>,
+        consensus_controller: Box<dyn ConsensusController>,
+        pool_controller: Box<dyn PoolController>,
+        receiver_network: Receiver<(PeerId, u64, Vec<u8>)>,
```

Comment:
use the type : PeerMessageTuple (it can be renamed)

Reply:
Done